.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+    // Allow `rust-analyzer` and `cargo` to run simultaneously.
+    // This uses a heavy amount of storage space, so consider commenting it out.
```

```suggestion
    // This uses extra storage space, so consider commenting it out.
```
It's not that heavy, as rust-analyzer doesn't do full builds.

---

I'll take your word for it, I haven't tried this setting myself yet.